src/dashboard/frontend/src/components/chat/ContextWindowMeter.module.css

```
@@ -71,7 +71,7 @@
   color: var(--muted-foreground);
   font-variant-numeric: tabular-nums;
   /* Use the SF Mono stack so digits inside the tiny ring stay aligned. */
-  font-family: "SF Mono", "SFMono-Regular", Consolas, "Liberation Mono", monospace;
+  font-family: var(--font-mono), "SF Mono", "SFMono-Regular", Consolas, "Liberation Mono", monospace;
```

Stylelint (17.14.1)

[error] Expected no quotes around "SFMono-Regular" (font-family-name-quotes)

(font-family-name-quotes)